<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="speedGrid">
     <item row="2" column="0">
      <widget class="QSlider" name="speedSlider">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="speedLabel">
           <property name="text">
            <string>Speed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>100 not recommended</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLCDNumber" name="speedLCD"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="timeGrid">
     <item row="1" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLCDNumber" name="minLCD">
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="secLCD">
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="millisecLCD">
         <property name="digitCount">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="timeName">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Timer:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="comboLabel">
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Pick your algorithms:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Bubble Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Selection Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Insertion Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Merge Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quick Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Heap Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Counting Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shell Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Radix Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bucket Sort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Intro Sort</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearAlgoButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>160</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Minutes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Seconds</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Milliseconds</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>1191</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="chartsGrid"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
